README.md

```diff
@@ -28,6 +28,10 @@ ALTER USER postgres PASSWORD 'postgres';
 CREATE DATABASE hexweb_dev;
 GRANT ALL PRIVILEGES ON DATABASE hexweb_dev TO postgres;
 ALTER USER postgres WITH SUPERUSER;
+
+-- if you also want to setup the test database
+CREATE DATABASE hexweb_test;
+GRANT ALL PRIVILEGES ON DATABASE hexweb_test TO postgres;
```

Comment:
not really related to this line, just a note:

I just remembered that sometimes I had issues with tests around midnight, so we could also add this:

```sql
ALTER DATABASE hex_test SET timezone TO 'UTC';
```

this is because my local database was by default set up with my local timezone. If it's worth to add, alternatively it could go into a migration. WDYT @ericmj?